test/services/pilot/create-asset-client/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+resources:
+- create_asset_client.yaml
+namespace: fybrik-system
+
+images:
+- name: create-asset-clt
+  newName: ghcr.io/rohithdv/create-asset-client
```

Comment:
Should not point to a personal account

Reply:
fixed. Added a placeholder for the github username. Please review. Thanks.